demo/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<!--
+    @license
```

Comment:
It was changed in https://github.com/PolymerLabs/seed-element/commit/39845f28720ca8ea99f2c63b86fa147c9d6526c5